website/blog/2023-08-03-adding-custom-query.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Adding a custom query
```

Comment:
```suggestion
title: Working with custom queries
```

Reply:
Yes, absolutely! 😄 Coming up with a good title on this one was killing me 😅